### ISSUE TITLE
npm

### ISSUE THREAD

**Issue by JohnnyTheTank** [Themercee/EasyCommands#1]
Hello, 

you could use npm as package manager to include your dependencies, like bootstrap, jquery, angular, bricklayer and angular-bricklayer.

do you need help?

**Comment by Themercee** [comment]
Yes if you want, it would be appreciate! I just switch to Bootstrap, before I was using HTML Kickstarter :P

**Comment by Themercee** [comment]
@JohnnyTheTank  I am trying to use the bricklayer module you did for angular, but can't figure out how to import and use it. When I try, I get an injector module error or the process renderer stay white and I can't open the dev tools. Some help would be really nice!

**Comment by JohnnyTheTank** [comment]
Hello @Themercee. I have no experience in node.js. But I will take a look this evening!
